### DISCUSSION TITLE
Global form sizing

### DISCUSSION THREAD

**Discussion by David-Maran** [twbs/bootstrap#36649] [Ideas]
Version 5 and no solutions for a global formatting to the form for the [size](https://getbootstrap.com/docs/5.2/forms/form-control/#sizing) size of the elements? like <form class="form-sm">

**Comment by WinterSilence** [comment]
[nope and not planed](https://github.com/twbs/bootstrap/issues/35896) :(

**Comment by mdo** [comment]
What do you mean by global formatting and sizing?

**Comment by David-Maran** [comment]
Sorry I missed the code tag so you can't read it. It was
Like: 
```html
<form class="form-sm"> 
```
Sizing all elements in one place, in the form tag.
